src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2537,69 +2538,40 @@ namespace cryptonote
   {
     struct request_t
     {
-      uint64_t height; // The height to query the quorum state for.
+      std::vector<uint64_t> heights; // Array of heights to query the quorums for, omit the height to request the latest quorum
```

Comment:
My comments from #708 apply here, as well: needing to pass in a vector of heights is a noticeably worse JSON RPC interface than being able to pass in a range when needing a range is a common use case.  I can see some utility here of being able to specifying individual heights (unlike in #708) but losing the range interface is a significant regression in the JSON RPC interface for getting quorum states.

I would suggest either keeping the range as is (and allowing `end` to be omitted to get just the `start_height`), or amending to allowing either a range or a list of heights, such as:
```
'start_height': 12345,
'end_height': 12456
```
or
```
'heights': [12345, 13579, 20000]
```
(This could even support both at once internally but just appending the range of values to the `heights` array).